met-web/src/components/survey/create/CreateOptions.tsx

```diff
@@ -94,6 +100,30 @@ export const CreateOptions = () => {
                     helperText={getErrorMessage()}
                 />
             </Grid>
+            <Grid item xs={6}></Grid>
+            <Grid item xs={6}>
+                <MetLabel sx={{ marginBottom: '2px' }}>Select Survey Type</MetLabel>
+                <Autocomplete
+                    id="survey-type-selector"
+                    options={[SurveyType.Single, SurveyType.Multi]}
```

Comment:
Autocomplete is overkill for this. We already know it's either single or multi. Toggle doesn't work? or if drop down is needed, I still prefer toggle, a simple textfield with select prop should work no?